### DISCUSSION TITLE
Hide or disable menu items

### DISCUSSION THREAD

**Discussion by Ultil** [getredash/redash#6264] [Q&A] [answered]
Hello,

Is there a way to disabel menu items, such as Alerts?

<img width="433" alt="image" src="https://github.com/getredash/redash/assets/140156248/e5a94e24-8cb3-4047-88b8-95ca94aa7127">

We wish to disable this feature.

Thank you in advance!

**Comment by gaecoli** [comment, marked as answer]
Now, there is no featute to hide `Alert` module.

**Comment by Ultil** [comment]
Thank you for the response
